lib/model.php

```diff
@@ -524,6 +534,9 @@ public function __set( $property, $value ) {
 		if ( $value !== null && \in_array( $property, $this->int_columns, true ) ) {
 			$value = (string) $value;
 		}
+		if ( $value !== null && \in_array( $property, $this->float_columns, true ) ) {
+			$value = (string) $value;
+		}
```

Comment:
Note for code reviewer: I decided to duplicate the lines above a bit, since in my opinion it is more readable than adding a separate condition in the previous `if` statement:
```php
if ( $value !== null && ( \in_array( $property, $this->int_columns, true ) || \in_array( $property, $this->float_columns, true ) ) ) {
	$value = (string) $value;
}
```